isic/core/tests/test_api_collection.py

```diff
@@ -84,10 +84,12 @@ def test_core_api_collection_populate_from_search(
 
     with django_capture_on_commit_callbacks(execute=True):
         r = authenticated_client.post(
-            f"/api/v2/collections/{collection.pk}/populate-from-search/", {"query": "sex:male"}
+            f"/api/v2/collections/{collection.pk}/populate-from-search/",
+            {"query": "sex:male"},
+            content_type="application/json",
```

Comment:
Are we ok with having to add this to every test request with a JSON body, or should we make our own subclass of Django's test `Client`?

Reply:
A: We should use `ninja.testing.TestClient`

Reply:
FWIW I think using the `django.test.TestClient` is preferable until [this issue](https://github.com/vitalik/django-ninja/issues/312) is resolved.